tests/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -298,12 +298,6 @@ def on_validation_start(self, _, pl_module):
                 pl_module, "on_validation_start", on_steps=[False], on_epochs=[True], prob_bars=self.choices
             )
 
-        def on_epoch_start(self, trainer, pl_module):
```

Comment:
These tests testing the logging output should stay

You can use `pytest.deprecated_call` to avoid the error